app/src/components/market/sections/market_verify/market_verify.tsx

```diff
@@ -108,12 +118,13 @@ const MarketVerifyWrapper: React.FC<Props> = (props: Props) => {
         </Button>
         <Button
           buttonType={ButtonType.primaryAlternative}
-          disabled={loading || typeof selection !== 'number' || !ovmInstance}
+          disabled={loading || typeof selection !== 'number' || !ovmAddress || verificationState != 1}
```

Comment:
@KadenZipfel well i set it like that because if state is other than 1 that means that verification can't be requested i found bug that when you request verification it will change text but you can still click on request verification and it will lag out when you click it...this solves it..